pkg/kubectl/cmd/portforward/portforward.go

```diff
@@ -210,6 +212,11 @@ func convertPodNamedPortToNumber(ports []string, pod corev1.Pod) ([]string, erro
 
 			containerPortStr = strconv.Itoa(int(containerPort))
 		}
+		ln, err := net.Listen("tcp", ":"+localPort)
+		if err != nil {
+			return nil, err
```

Comment:
I'm curious about the error message here when the port is in use. Maybe we could wrap the error with additional information to clarify what happens here.

/priority backlog
/ok-to-test

Reply:
Thanks for reviewing this! I was thinking the same thing, but the error message ends up being very helpful:
`error: listen tcp :3306: bind: address already in use`